.github/workflows/deploy.yaml

```diff
@@ -3,7 +3,7 @@ name: Deploy to Heroku
 on:
   push:
     branches:
-      - main
+      - "**" # Matches all branch and tag names.
```

Comment:
just got feedback from abby and george, you can just comment out line 6. so instead of adding:

- "**" # Matches all branch and tag names.

you can just add a hash at the beginning of line 6. that's probably better but they both do the same thing

#   - main